### PR TITLE
Feature/dfe 765 print page

### PR DESCRIPTION
**Pull request by bennettstuart**
Spruced up the print page styling to make the publications look more like documents when printed

### CODE REVIEW

**Comment by jamescutts** [comment]
@bennettstuart npm run test:ci failed

File: src/explore-education-statistics-common/src/styles/_print.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.dfe-print-hidden {
+  @include print-hide();
+}
+
+@mixin print-unbreakable() {
+  page-break-inside: avoid;
+}
+.dfe-print-unbreakable {
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/styles/_print.scss]: Nitpick but unbreakable sounds weird, `dfe-print-break-avoid` is probably more inline with the CSS property itself

**Comment by jamescutts** [comment]
@bennettstuart merge conflicts need resolving then should be good

**Comment by jamescutts** [comment]
npm run test:ci

```
Summary of all failing tests
FAIL src/components/__tests__/AccordionSection.test.tsx
  ● AccordionSection › renders correctly with required props

    expect(received).toMatchSnapshot()

    Snapshot name: `AccordionSection renders correctly with required props 1`

    - Snapshot
    + Received

    @@ -1,6 +1,6 @@
    - <div class="govuk-accordion__section"
    + <div class="govuk-accordion__section section"
           role="presentation"
      >
        <div class="govuk-accordion__section-header">
          <h2 class="govuk-accordion__section-heading">
            <button aria-expanded="false"

      13 |     expect(getByText('Test heading')).toBeDefined();
      14 |     expect(getByText('Test content')).toBeDefined();
    > 15 |     expect(container.innerHTML).toMatchSnapshot();
         |                                 ^
      16 |   });
      17 | 
      18 |   test('renders with different heading size', () => {

      at Object.toMatchSnapshot (src/components/__tests__/AccordionSection.test.tsx:15:33)

  ● AccordionSection › renders with different heading size

    expect(received).toMatchSnapshot()

    Snapshot name: `AccordionSection renders with different heading size 1`

    - Snapshot
    + Received

    @@ -1,6 +1,6 @@
    - <div class="govuk-accordion__section"
    + <div class="govuk-accordion__section section"
           role="presentation"
      >
        <div class="govuk-accordion__section-header">
          <h3 class="govuk-accordion__section-heading">
            <button aria-expanded="false"

      25 |     expect(container.querySelector('h2')).toBeNull();
      26 |     expect(container.querySelector('h3')).not.toBeNull();
    > 27 |     expect(container.innerHTML).toMatchSnapshot();
         |                                 ^
      28 |   });
      29 | 
      30 |   test('renders with caption', () => {

      at Object.toMatchSnapshot (src/components/__tests__/AccordionSection.test.tsx:27:33)

  ● AccordionSection › renders with caption

    expect(received).toMatchSnapshot()

    Snapshot name: `AccordionSection renders with caption 1`

    - Snapshot
    + Received

    @@ -1,6 +1,6 @@
    - <div class="govuk-accordion__section"
    + <div class="govuk-accordion__section section"
           role="presentation"
      >
        <div class="govuk-accordion__section-header">
          <h2 class="govuk-accordion__section-heading">
            <button aria-expanded="false"

      36 | 
      37 |     expect(getByText('Some caption text')).toBeDefined();
    > 38 |     expect(container.innerHTML).toMatchSnapshot();
         |                                 ^
      39 |   });
      40 | });
      41 | 

      at Object.toMatchSnapshot (src/components/__tests__/AccordionSection.test.tsx:38:33)

```

**Comment by bennettstuart** [comment]
I rebased from master here, which is a bit gross, will merge master in future

**Pull request merged**